src/Drawer/Drawer.js

```diff
@@ -71,7 +71,7 @@ export default class Drawer extends Component {
     styleManager: PropTypes.object.isRequired,
   };
 
-  getSlideDirection(anchor) {
+  static getSlideDirection(anchor) {
```

Comment:
A new eslint rule `warn` on class methods not using the class state. I.e that could be made static.
My main concern is regarding, how we can test those fonctions.